pkg/config/config_test.go

```diff
@@ -306,17 +306,21 @@ func Test_checkFilePerms(t *testing.T) {
 	}
 	f1, err := ioutil.TempFile(os.TempDir(), "prefix-")
 	if err != nil {
-		t.FailNow()
+		t.Errorf("Cannot create 1st temp file: %s", err)
```

Comment:
`t.Errorf` does not stop the execution of the test. And if `f1` is nil, then we'd get a panic. We should use `t.Fatalf` no?